libraries/chain/include/eosio/chain/abi_serializer.hpp

```diff
@@ -498,16 +523,28 @@ namespace impl {
          out(name, std::move(mvo));
       }
 
+
+      /**
+       * overload of to_variant_object for signed_block, support old signed_block_v0 format
+       */
+      template<typename Resolver>
+      static void add( mutable_variant_object &out, const char* name, const signed_block& block, Resolver resolver, abi_traverse_context& ctx )
+      {
+         // TODO: this could be faster by going directly to variant
+         auto sb_v0 = block.to_signed_block_v0();
```

Comment:
This will fail if any context free data has been pruned.  Given that you are handling this case for packed_transaction, I think you should also handle it here in the same way.

Reply:
Updated to correctly serialize to `signed_block_v0` format.